boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -49,28 +49,28 @@ public void testForNewLinkageErrors()
     Path bomFile = Paths.get("../cloud-oss-bom/pom.xml");
     Bom bom = Bom.readBom(bomFile);
 
-    ImmutableSetMultimap<SymbolProblem, ClassFile> oldProblems =
+    ImmutableSetMultimap<LinkageProblem, ClassFile> oldProblems =
```

Comment:
This probably doesn't need to be a multimap anymore since LinkageProblem points to the class file.

Reply:
Yes, updated.